txjuju/config.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2016 Canonical Limited.  All rights reserved.
+
+from collections import namedtuple
+
+
+class Config(object):
```

Comment:
I thought we were missing Config class unit tests but it seems that test coverage comes in pull request 4 so I'll review that too.